core-libs/setup/ssr/logger/transformers/default-transformers.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { DefaultTraceparentTransformer } from './traceparent/default-traceparent-transformer';
+
+export interface TransformerConfig {}
+
+export const defaultTransformersConfig: TransformerConfig = {
```

Comment:
word `transformer` is too general in the context of public api of `@spartacus/setup/ssr`.
What about `LogTransformer`? 

LogTransformerConfig
defaultLogTransformerConfig
DefaultTraceparentLogTransformer